todo/Services/TodoTaskService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Todo.Models;
+using Todo.DTOs;
+using Microsoft.EntityFrameworkCore.Storage;
+
+namespace Todo.Services
+{
+    public class todoTaskService : ITodoTaskService
+    {
+        private readonly DataAccessContext _context;
+        private readonly IMapper _mapper;
+        public todoTaskService(DataAccessContext context, IMapper mapper)
+        {
+            _context = context;
+            _mapper = mapper;
+        }
+        const bool Done = false;
+        public async Task<TodoTaskDTO> AddTodoTaskAsync(TodoTaskDTO model)
+        {
+            var newTask = _mapper.Map<TodoTask>(model);
+            _context.TodoTasks!.Add(newTask);
+            await _context.SaveChangesAsync();
+
+            return model;
+        }
+
+        public async Task CompleteTaskAsync(List<int> id)
+        {
+            var task = await _context.TodoTasks.Where(c 
+                => id.Contains(c.TaskId)).ToListAsync();
+            foreach (var x in task)
+            {
+                x.Status = Done;
+            }
+            _context.UpdateRange(task);
+            await _context.SaveChangesAsync();
+        }
+
+        public async Task DeleteTodoTaskAsync(int id)
+        {
+            var deleteTask = await _context.TodoTasks!.FindAsync(id);
+            if(deleteTask != null)
+            {
+                _context.TodoTasks!.Remove(deleteTask);
+                await _context.SaveChangesAsync();
+            }
+        }
+
+        public async Task<List<TodoTaskDTO>> GetAllTodoTasksAsync()
+        {
+            var task = await _context.TodoTasks!.ToListAsync();
```

Comment:
```suggestion
            var task = await _context.TodoTasks!.ToListAsync();
```
sao vẫn để ! ở đây nhỉ ?